lib/ar_query_matchers/queries/load_counter.rb

```diff
@@ -15,24 +15,17 @@ def self.instrument(&block)
       end
 
       class LoadQueryFilter < Queries::QueryFilter
-        # Matches named SQL operations like the following:
-        # 'User Load'
-        MODEL_LOAD_PATTERN = /\A(?<model_name>[\w:]+) (Load|Exists)\Z/.freeze
-
         # Matches unnamed SQL operations like the following:
-        # "SELECT COUNT(*) FROM `users` ..."
-        MODEL_SQL_PATTERN = /SELECT .* FROM [`"](?<table_name>[^`"]+)[`"]/.freeze
-
-        def filter_map(name, sql)
-          # First check for a `SELECT * FROM` query that ActiveRecord has
-          # helpfully named for us in the payload
-          match = name.match(MODEL_LOAD_PATTERN)
-          return ModelName.new(match[:model_name]) if match
```

Comment:
**Note**: The model this is finding is already present in the sql printout. Including it would cause a duplicate count (or we'd have to potentially reconcile the two sources).

Comment from ZP relevant to this: https://github.com/Gusto/zenpayroll/pull/96506#discussion_r639432314